Vagrantfile

```diff
@@ -41,6 +41,13 @@ Vagrant.configure(2) do |config|
   # the elasticsearch project called vagrant....
   config.vm.synced_folder '.', '/vagrant', disabled: true
   config.vm.synced_folder '.', '/elasticsearch'
+  # TODO: make these syncs work for windows!!!
+  config.vm.synced_folder "#{Dir.home}/.vagrant/gradle/caches", "/root/.gradle/caches",
```

Comment:
No it needs to be the root user's .gradle dir, since we run these tests as root (so they can eg install packages).

Reply:
My question is what is `#{Dir.home}/.vagrant/gradle/caches`? Is that the "source"?

Reply:
Yes, that is the host path. I put it in separate directory from gradle's caches based on some work long ago where I had issues with multiple builds running at the same time, but I'm happy to try it again if you think it will work.